src/thunderbots/software/ai/world/world.cpp

```diff
@@ -23,7 +23,10 @@ World::World(const Field &field, const Ball &ball, const Team &friendly_team,
       ball_(ball),
       friendly_team_(friendly_team),
       enemy_team_(enemy_team),
-      game_state_()
+      game_state_(),
+      // Store a small buffer of previous refbox game states so we can filter out noise
+      // Do not make this value smaller than 3
```

Comment:
See my comment below, I think we can remove this restriction

Reply:
done